syntax = "proto3";

package response_models;
import "player-position-response-model.proto";


// PlayerPositionResponseModel
message PlayersPositionsResponseModel {
	repeated response_models.PlayerPositionResponseModel players = 1;
}
